sem_covid/entrypoints/etl_dags/eurlex_covid19.py

```diff
@@ -435,16 +435,16 @@ def make_request(query):
 
 
 def clear_bucket():
-    minio = MinioAdapter(config.EU_CELLAR_BUCKET_NAME, config.MINIO_URL, config.MINIO_ACCESS_KEY,
-                         config.MINIO_SECRET_KEY)
-    minio.empty_bucket(object_name_prefix=None)
-    minio.empty_bucket(object_name_prefix=RESOURCE_FILE_PREFIX)
-    minio.empty_bucket(object_name_prefix=TIKA_FILE_PREFIX)
+    minio = MinioObjectStorage(config.EU_CELLAR_BUCKET_NAME, config.MINIO_URL, config.MINIO_ACCESS_KEY,
+                               config.MINIO_SECRET_KEY)
+    minio.clear_storage(object_name_prefix=None)
```

Comment:
this method name is misleading; 
It deletes objects at a particular "path"/"prefix"

Reply:
Nor the old name was clear, it was as misleading